loops-and-arrays.js

```diff
@@ -56,6 +64,8 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
-
+for (let i = 0; i < 5; i++) {
+    console.log(squares[i]*squares[i]*squares[i])
```

Comment:
Wist je dat je ook het volgende kan doen?
```suggestion
    console.log(squares[i]**3)
```